### PR TITLE
Add Helm Chart Network Service and annotations

### PR DESCRIPTION
**Pull request by PatStLouis**

### CODE REVIEW

File: charts/tdw-server/templates/postgresql/networkpolicy.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  podSelector:
+    matchLabels:
+      {{- include "postgresql.selectorLabels" . | nindent 6 }}
+  ingress:
+    - from:
+        - namespaceSelector:
+            matchLabels:
+              {{- toYaml .Values.networkPolicy.ingress.namespaceSelector | nindent 14 }}
+  policyTypes:
+    - Ingress
```
Comment (esune) [inline charts/tdw-server/templates/postgresql/networkpolicy.yaml]: I don't think we need an ingress for postgresql, do we?
The resource will likely sit protected from the outside world.

File: charts/tdw-server/templates/server/ingress.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ kind: Ingress
 metadata:
   name: {{ include "server.fullname" . }}
   labels:
-    {{- include "server.labels" . | nindent 4 }}
+    {{- toYaml .Values.server.labels | nindent 4 }}
```
Comment (esune) [inline charts/tdw-server/templates/server/ingress.yaml:7]: I suggest including the default server labels like you had it before AND add custom labels, if any are defined. You will likely need to update the template to use `.Values.ingress.labels` for that, based on the values file structure.

**Review by esune — CHANGES_REQUESTED** [review]
A couple of tweaks/suggestions.
Also any specific reason to remove the chart linting action?

File: charts/tdw-server/templates/postgresql/networkpolicy.yaml (hunk 1 above)
Comment (PatStLouis) [reply charts/tdw-server/templates/postgresql/networkpolicy.yaml]: I believe the ingress was required to allow the server to communicate with the postgresql service. At this point I will take your recommendations as this is something that depends on which security policies are in place in open shift, just make an edit with your recommendations

File: charts/tdw-server/templates/server/ingress.yaml (hunk 2 above)
Comment (PatStLouis) [reply charts/tdw-server/templates/server/ingress.yaml:7]: linting was giving me an issue on that one, this is why I had removed them

File: charts/tdw-server/templates/postgresql/networkpolicy.yaml (hunk 1 above)
Comment (PatStLouis) [reply charts/tdw-server/templates/postgresql/networkpolicy.yaml]: first network policy has been removed

File: charts/tdw-server/templates/server/ingress.yaml (hunk 2 above)
Comment (PatStLouis) [reply charts/tdw-server/templates/server/ingress.yaml:7]: added the include statement back in

**Comment by PatStLouis** [comment]
@esune it was giving me headaches on the external dependencies. 

This was the error:
`Error: no repository definition for https://charts.bitnami.com/bitnami/, [https://charts.bitnami.com/bitnami/.](https://charts.bitnami.com/bitnami/) Please add the missing repos via 'helm repo add'`

**Review by esune — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**